components/DropdownLight/DropdownLight.jsx

```diff
@@ -23,20 +35,109 @@ const DropdownSelect = styled.select`
   }}
 `;
 
-const DropdownLight = ({ disabled, items, theme, placeholder }) => (
-  <>
-    <DropdownSelect disabled={disabled} theme={theme} defaultValue="">
-      <option value="" hidden>
-        {placeholder}
-      </option>
-      {items.map(item => (
-        <option value={item?.value || item} key={item?.value || item}>
-          {item?.label || item}
-        </option>
-      ))}
-    </DropdownSelect>
-  </>
-);
+const Input = styled.input`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+  text-align: left;
+`;
+
+const ArrowIcon = styled(Icon)`
+  display: inline-block;
+  pointer-events: none;
+  width: 24px;
+
+  ${({ selectedItem }) =>
+    !selectedItem &&
+    `
+    color: inherit;
+  `}
+`;
+
+const ButtonField = styled.button`
+  width: 100%;
+  margin-top: 0px;
+  padding: 8px 16px;
+  border-radius: 4px;
+  box-sizing: border-box;
+  font-size: ${baseFontSize};
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-align: left;
+
+  -webkit-transition: all 0.2s ease-in-out;
+  transition: all 0.2s ease-in-out;
+  background-color: #ffffff;
+  border: 2px solid ${colors.neutral['500']};
+  color: ${colors.neutral['700']};
+
+  :hover {
+    border-color: ${colors.primary['700']};
+    box-shadow: 0px 3px 1px -2px rgb(18 80 196 / 20%),
+      0px 2px 2px 0px rgb(18 80 196 / 14%), 0px 1px 5px 0px rgb(18 80 196 / 12%);
+  }
+
+  :focus {
+    border-color: ${colors.primary['700']};
+    box-shadow: 0px 3px 1px -2px rgb(18 80 196 / 20%),
+      0px 2px 2px 0px rgb(18 80 196 / 14%), 0px 1px 5px 0px rgb(18 80 196 / 12%);
+  }
+`;
+
+const OptionItem = styled.li`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  cursor: pointer;
+  min-height: 42px;
+  padding: 8px 16px;
+  :hover {
+    background-color: ${colors.neutral['100']};
+  }
+`;
+
+const DropdownLight = ({ disabled, items, theme, placeholder }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedItem, setSelectedItem] = useState('');
+  const [itemLabel, setItemLabel] = useState(placeholder);
+
+  const handleClose = item => {
+    setIsOpen(false);
+    setSelectedItem(item?.value || item);
+    setItemLabel(item?.label || item);
+  };
+
+  return (
+    <>
+      <Input type="hidden" value={selectedItem} />
+
+      <ButtonField onClick={() => setIsOpen(!isOpen)}>
+        {itemLabel}
+        <ArrowIcon
+          name={isOpen ? 'arrow_drop_up' : 'arrow_drop_down'}
+          theme={theme}
+        />
+      </ButtonField>
+
+      {isOpen && (
+        <DropdownSelect disabled={disabled} theme={theme}>
+          {items.map(item => (
+            <OptionItem
+              key={item?.value || item}
+              onClick={() => handleClose(item)}
+            >
+              {item?.label || item}
+            </OptionItem>
```

Comment:
When a list item is selected there is a highlight and an icon on it, see the implementation of the old dropdown

Reply:
I did a review and noticed that the checkbox is missing, but as the control component is in the next task I will complete the logic in it. (KeyPress QTM-189 )

![image](https://user-images.githubusercontent.com/37362664/213596478-7e9e8101-e5f8-4a62-ae3a-752697c9a263.png)